tcms_tenants/checks/tenants_env.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from django.core import checks
+
+
+@checks.register
+def tenants_env_check(app_configs, **kwargs):
+    errors = []
+    if os.environ.get('KIWI_TENANTS_DOMAIN') is None:
+        errors.append(
+            checks.Error(
+                level='CRITICAL',
+                msg='KIWI_TENANTS_DOMAIN environment variable is not set!',
+                hint='You need this variable set to run tenancy enabled system.',
```

Comment:
Change to something like `This variable is mandatory, see https://github.com/kiwitcms/tenants#installation`